### ISSUE TITLE
[BUG] Wired Pro Controller inputs going crazy randomly

### ISSUE THREAD

**Issue by Ledrac** [Davidobot/BetterJoy#566]
**Running the 6.4a version.** 
While BetterJoy's running, after some time (can vary a lot), the icon that shows the Pro Controller's state turns into red immediately, making some unintended inputs like:
• Random analog inputs;
• Random second analog inputs;
• Second analog inputs while you press a button;

Everything just goes back to normal sometimes, and then comes back. I've tried to reinstall the drivers, restarting the computer, running as administrator, plugging off and plugging in the controller to the USB port again. 
The controller is in a perfect working condition and doesn't have any problems on Switch.

Any ideas?

**Comment by jotacee98** [comment]
Did you find a fix for this? Also happens to me

**Comment by Ledrac** [comment]
> Did you find a fix for this? Also happens to me
Nope, but, for some reason, this  glitch just stopped happening to me just a few days ago, so...

**Comment by TheToto** [comment]
Related to https://github.com/Davidobot/BetterJoy/issues/584
